microsoft/botframework-solutions
# [Enterprise Bot Template][TypeScript] Update template with latest canges

## Description
- Update template with latest changes from sample

## Related Issue
Fixes #630

## Testing Steps
1. Install generator with `npm link`
1. Execute generator with `yo botbuilder-enterprise`
1. It should behave as the sample from [Enterprise Bot](https://github.com/Microsoft/AI/tree/master/templates/Enterprise-Template/src/typescript/enterprise-bot)

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have added or updated the appropriate unit tests~
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
